pennylane/math/quantum.py

```diff
@@ -606,3 +606,158 @@ def _compute_mutual_info(
     )
 
     return vn_entropy_1 + vn_entropy_2 - vn_entropy_12
+
+
+def fidelity(state0, state1, check_state=False, c_dtype="complex128"):

```

Comment:
Function doesn't seem to appear in the docs.

Reply:
I added it to rst 👍